manifests/firewall.pp

```diff
@@ -56,7 +56,7 @@
     firewall { '100 forward to perfSONAR ipv6':
       chain    => 'INPUT',
       jump     => 'perfSONAR',
-      provider => 'ip6tables',
+      protocol => 'ip6tables',
```

Comment:
```suggestion
      protocol => 'IPv6',
```

Reply:
Based on firewall module migration to 7.0.0.  Says `ip6tables` will work but docs now use `IPv6`